vision.org

```diff
@@ -0,0 +1,32 @@
+#+TITLE: The Future of the ediTDor
+#+AUTHOR: Pedram Hadjian
+
+* Vision
+
+The Web of Things enables us to represent operational digital twins in an interoperable way. The main problem for IoT developers is the access to and understanding of data. Data in the context of IoT can broadly be split into two different categories:
+
+1. Timeseries Data: data coming from sensors, oftentimes called signals, runtime data, samples or others
+
+2. Contextual Data: this type of data answers questions about types of devices, interpretation of timeseries values, topology of assets and locations, taxonomies and more. Sometimes this type of data is called the semantics.
+
+We do not have a consistent way to work with these kinds of data. While normal software developers happily whip out VSCode, IntelliJ, emacs or vim, install a language server and make use of a myriad of libraries to get things done, IoT developers struggle to even access and understand data, let alone deliver value on top of it.
+
+In this open source project, we set out to change this. The vision is to bridge the gap between the two scenarios. We will focus on the developer experience and if we succeed, an IoT developer will be able to create or better just fetch device twins, model the context, write code and deploy to production.
+
+The vision is grand, but we need to start small. At Siemens we published the open source [[https://github.com/eclipse/editdor][ediTDor]] project, which helps to write and syntactically validate TMs/TDs. We will pick this up and drive it towards an awesome Digital Twin development environment.
```

Comment:
Siemens did not publish the EdiTDor open source project, Eclipse Foundation did, to which people with Siemens affiliation have contributed to. Since https://projects.eclipse.org/projects/iot.editdor/who does not even show Siemens (due to Committers not choosing their affiliation), this should be reworded.

Reply:
Sure, it's hard to see that for me, since I have the inside view of this project, but you are right: there is no Siemens affiliation mentioned and this is actually what we want. We want a community driven project. Will change that.

Reply:
https://github.com/web-of-things-open-source/proposal/commit/1abdb344aad1f547f36bbcf116c1466007d25a92
- Replaced Siemens with Eclipse Foundation